ts/src/test/static/request/bitget.json

```diff
@@ -1163,11 +1163,11 @@
           {
             "description": "spot trading fees",
             "method": "fetchTradingFees",
-            "url": "https://api.bitget.com/api/v2/spot/public/symbols?0=BTC%2FUSDT",
+            "url": "https://api.bitget.com/api/v2/spot/public/symbols?symbol=BTCUSDT",
             "input": [
```

Comment:
actually I will remove these completely fetchTradingFees does not accept symbol so it is not expected ot provide one